CHANGELOG.md

```diff
@@ -70,3 +70,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Bump python to v3.11.4
 - Pin Dash version over 2.0 and under 2.10
 - Remove Selenium and Werkzeug pin
+
+## [0.3.1] - 2023-09-22
```

Comment:
💡 

What about packaging / publishing a new release when more substantial changes have been made to the library to simplify our deployment pipeline?